__tests__/Place/place.test.ts

```diff
@@ -0,0 +1,124 @@
+import axios from "axios";
+import { Place, Rating } from "../../src/db/types";
+import { dbConnection } from "../../src/db/mongoConnection";
+
+describe("Place database functions", () => {
+	it("/addPlace adds place", async () => {
+		const { _db, _connection } = await dbConnection();
+		await _db.dropDatabase();
+		await _connection.close();
+
+		let place!: Place;
+		let responseStatus!: number;
+		try {
+			// eslint-disable-next-line
+			const { data, status } = await axios.post("http://localhost:3030/addPlace", {
+				_id: "faketestid1",
+			});
```

Comment:
```suggestion
			const { data, status } = await axios.post("http://localhost:27017/addPlace", {
				_id: "faketestid1",
			});
```

Check this out: https://github.com/marketplace/actions/mongodb-in-github-actions

> This GitHub Action starts a MongoDB server or MongoDB replica set. By default, the MongoDB server is available on the default port 27017.

Reply:
Ah, yeah you're right. I was going to change the port once I'm ready since I'm still testing as I write (I'm only like 50% done with the tests I want to write). I'll definitely make sure to go back and change these!

Reply:
Oh wait, the port here is not the mongo port, its the port where the express server is running @carpenterd777

Reply:
no I think you're right, we shouldn't interfere with the express server port. we'll have to think of a way to run the express server using a github action too